src/md4c-html.c

```diff
@@ -265,6 +265,13 @@ render_attribute(MD_HTML* r, const MD_ATTRIBUTE* attr,
     }
 }
 
+static void
+render_mention_link(MD_HTML* r, const MD_SPAN_MENTION_DETAIL* det)
+{
+    RENDER_VERBATIM(r, "<x-mention data-target=\"");
+    render_entity(r, det->text, det->size, render_html_escaped);
```

Comment:
Should likely call `render_verbatim()` instead. No entity translation should place here I guess.